app/screens/settings-screen/settings-screen.tsx

```diff
@@ -199,75 +202,67 @@ export const SettingsScreen: React.FC = () => {
         })
       },
       chevronLogo: lightningAddress ? "copy" : undefined,
-      enabled: isAuthed,
-      greyed: !isAuthed,
+      enabled: isAtLeastLevelOne,
+      greyed: !isAtLeastLevelOne,
     },
     {
       category: LL.SettingsScreen.addressScreen(),
       icon: "custom-receive-bitcoin",
       id: "address",
       action: () => navigation.navigate("addressScreen"),
-      enabled: isAuthed && Boolean(lightningAddress),
-      greyed: !isAuthed || !lightningAddress,
-    },
-    {
-      category: LL.common.transactionLimits(),
-      id: "limits",
-      icon: "custom-info-icon",
-      action: () => navigation.navigate("transactionLimitsScreen"),
-      enabled: isAuthed,
-      greyed: !isAuthed,
+      enabled: isAtLeastLevelOne && Boolean(lightningAddress),
```

Comment:
did we said we will be going for isAtLeastLevelZero still here? it will enable those users to discover/use the POS

Reply:
You're saying you want to allow device accounts to create a lightning address?